web/src/main/java/org/mskcc/cbio/oncokb/api/pvt/PrivateGenomeNexusController.java

```diff
@@ -46,5 +55,54 @@ public ResponseEntity<TranscriptResult> getTranscript(
         return new ResponseEntity<>(transcriptResult, HttpStatus.OK);
     }
 
+    @ApiOperation(value = "", notes = "Fetch Genome Nexus variant info by HGVSg", response = GenomeNexusAnnotatedVariantInfo.class, responseContainer = "List")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "OK", response = GenomeNexusAnnotatedVariantInfo.class, responseContainer = "List"),
+        @ApiResponse(code = 400, message = "Error, error message will be given.", response = String.class)})
+    @RequestMapping(value = "/fetch-gn-variant-info/byHGVSg",
```

Comment:
Sorry about changing the url again, but it doesn't seem to be the case for oncokb-core to use dash in the url. Let's use /fetchGnVariants/byHGVSg